pandas/tests/resample/test_datetime_index.py

```diff
@@ -516,7 +516,7 @@ def test_upsample_with_limit(unit):
     tm.assert_series_equal(result, expected)
 
 
-@pytest.mark.parametrize("freq", ["5D", "10h", "5Min", "10s"])
+@pytest.mark.parametrize("freq", ["1D", "10h", "5Min", "10s"])
```

Comment:
no risk we're losing actual coverage here?

Reply:
I don't think so. From reading https://github.com/pandas-dev/pandas/pull/33939, I think 5D was arbitrarily chosen, so I chose 1D to have a smaller day frequency resampling.